facefusion/uis/components/webcam.py

```diff
@@ -27,15 +27,17 @@
 WEBCAM_START_BUTTON : Optional[gradio.Button] = None
 WEBCAM_STOP_BUTTON : Optional[gradio.Button] = None
 
-
-def get_webcam_capture() -> Optional[cv2.VideoCapture]:
+def get_webcam_capture(webcam_mode: WebcamMode = None) -> Optional[cv2.VideoCapture]:
 	global WEBCAM_CAPTURE
-
+	webcam_mode = get_ui_component('webcam_mode_radio')
+	webcam_url = get_ui_component('webcam_url_textbox')
 	if WEBCAM_CAPTURE is None:
 		if platform.system().lower() == 'windows':
```

Comment:
the conditions have to be smarter... utilize the new webcam_device and webcam_url method arguments

also make sure it does not create a local webcam instance first, just to override it with a remote 5 lines later